util/settings/settings.go

```diff
@@ -290,31 +274,6 @@ var (
 	}
 )
 
-func (ks *KustomizeSettings) GetOptions(source v1alpha1.ApplicationSource) (*v1alpha1.KustomizeOptions, error) {
```

Comment:
The behavior of this function is now handled by `getKustomizeBinaryPath` in reposerver/reposerver/repository.go.